components/form/FAutocompleteInput.vue

```diff
@@ -44,45 +47,44 @@ FField.FAutocompleteInput(
         hide-error-icon
         hide-hint
         @focus="e => handleFocus(e, openMenu)"
-        @blur="handleBlur"
-        @change="handleChange"
+        @blur="e => handleBlur(e, closeMenu)"
+        @change="e => emit('change', e)"
```

Comment:
for consistency, you could define a handler too
```suggestion
        @change="handleChange"
```